src/test/data/JsonSerializableXpireTest/typicalItemsXpire.json

```diff
@@ -3,17 +3,14 @@
   "items" : [ {
     "name" : "Milk",
     "expiryDate" : "01/02/2019",
-    "reminderThreshold" : "0",
-    "tags" : [ "drinks" ]
```

Comment:
JSON item model is
{
    "name: <name>
    "expireDate" : <date>
    "reminderThreshold" : <threshold>
    "tags" : <array of tags>
}

so there should be a field "tags"